concrete/attributes/topics/controller.php

```diff
@@ -134,9 +143,12 @@ public function setNodes($akTopicParentNodeID, $akTopicTreeID)
 
     public function createAttributeValue($nodes)
     {
-        $selected = array();
+        $selected = [];
         $this->load();
         $tree = Tree::getByID($this->akTopicTreeID);
+        if (is_array($nodes) && $this->akTopicAllowMultipleValues == false) {
+            $nodes = $nodes[0];
```

Comment:
If anything I'd just want to show an error instead of just going with the first item.